ReactiveUI/Cocoa/CommonReactiveSource.cs

```diff
@@ -278,30 +234,74 @@ void resetupAll(IReadOnlyList<TSectionInfo> newSectionInfo)
             //
             // TODO: Instead of listening to Changed events and then reseting,
             // we could listen to more specific events and avoid some reloads.
-            disp.Add(sectionChanging.Subscribe(_ => {
+            disp.Add(sectionChanging.ObserveOn(ImmediateScheduler.Instance).Subscribe(_ => {
```

Comment:
orly? You building in XS or VS? I feel like something's balked recently with XS, and now I can't even check for updates - just hangs every time.

I would expect the error you said if you are trying to build in VS. It's the problem I mentioned on Slack whereby VS expects references that XS does not. I just Ctrl+. them all in and make sure I don't commit those project changes. Very annoying.